include/audio/aseq.h

```diff
@@ -0,0 +1,1979 @@
+#define PITCH_AF2      23
+#define PITCH_A2       24
+#define PITCH_BF2      25
+#define PITCH_B2       26
+#define PITCH_C3       27
+#define PITCH_DF3      28
+#define PITCH_D3       29
+#define PITCH_EF3      30
+#define PITCH_E3       31
+#define PITCH_F3       32
+#define PITCH_GF3      33
+#define PITCH_G3       34
+#define PITCH_AF3      35
+#define PITCH_A3       36
+#define PITCH_BF3      37
+#define PITCH_B3       38
+#define PITCH_C4       39
+#define PITCH_DF4      40
+#define PITCH_D4       41
+#define PITCH_EF4      42
+#define PITCH_E4       43
+#define PITCH_F4       44
+#define PITCH_GF4      45
+#define PITCH_G4       46
+#define PITCH_AF4      47
+#define PITCH_A4       48
+#define PITCH_BF4      49
+#define PITCH_B4       50
+#define PITCH_C5       51
+#define PITCH_DF5      52
+#define PITCH_D5       53
+#define PITCH_EF5      54
+#define PITCH_E5       55
+#define PITCH_F5       56
+#define PITCH_GF5      57
+#define PITCH_G5       58
+#define PITCH_AF5      59
+#define PITCH_A5       60
+#define PITCH_BF5      61
+#define PITCH_B5       62
+#define PITCH_C6       63
+#define PITCH_DF6      64
+#define PITCH_D6       65
+#define PITCH_EF6      66
+#define PITCH_E6       67
+#define PITCH_F6       68
+#define PITCH_GF6      69
+#define PITCH_G6       70
+#define PITCH_AF6      71
+#define PITCH_A6       72
+#define PITCH_BF6      73
+#define PITCH_B6       74
+#define PITCH_C7       75
+#define PITCH_DF7      76
+#define PITCH_D7       77
+#define PITCH_EF7      78
+#define PITCH_E7       79
+#define PITCH_F7       80
+#define PITCH_GF7      81
+#define PITCH_G7       82
+#define PITCH_AF7      83
+#define PITCH_A7       84
+#define PITCH_BF7      85
+#define PITCH_B7       86
+#define PITCH_C8       87
+#define PITCH_DF8      88
+#define PITCH_D8       89
+#define PITCH_EF8      90
+#define PITCH_E8       91
+#define PITCH_F8       92
+#define PITCH_GF8      93
+#define PITCH_G8       94
+#define PITCH_AF8      95
+#define PITCH_A8       96
+#define PITCH_BF8      97
+#define PITCH_B8       98
+#define PITCH_C9       99
+#define PITCH_DF9      100
+#define PITCH_D9       101
+#define PITCH_EF9      102
+#define PITCH_E9       103
+#define PITCH_F9       104
+#define PITCH_GF9      105
+#define PITCH_G9       106
+#define PITCH_AF9      107
+#define PITCH_A9       108
+#define PITCH_BF9      109
+#define PITCH_B9       110
+#define PITCH_C10      111
+#define PITCH_DF10     112
+#define PITCH_D10      113
+#define PITCH_EF10     114
+#define PITCH_E10      115
+#define PITCH_F10      116
+#define PITCH_BFNEG1   117
+#define PITCH_BNEG1    118
+#define PITCH_C0       119
+#define PITCH_DF0      120
+#define PITCH_D0       121
+#define PITCH_EF0      122
+#define PITCH_E0       123
+#define PITCH_F0       124
+#define PITCH_GF0      125
+#define PITCH_G0       126
+#define PITCH_AF0      127
+
+// Hardcoded Instruments
+#define FONTANY_INSTR_SFX          126
+#define FONTANY_INSTR_DRUM         127
+// gWaveSamples[]
```

Comment:
What is this comment for?

Reply:
These instruments are implemented by the waveforms pointed to by `gWaveSamples`

Reply:
Wonder if this would be better as maybe like `gWaveSamples` Instruments, but not probably fine as is.